src/test/kotlin/CalculatorTest.kt

```diff
@@ -0,0 +1,35 @@
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import java.lang.IllegalArgumentException
+
+class CalculatorTest {
+    @DisplayName("계산기 테스트")
+    @Test
+    fun calculate() {
+        val calculator = Calculator()
+        val input = listOf<String>("4", "+", "4", "/", "2", "*", "2")
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val input = listOf("4", "+", "4", "/", "2", "*", "2")
```